Assets/VRTK/Scripts/VRTK_InteractGrab.cs

```diff
@@ -37,6 +37,8 @@ public class VRTK_InteractGrab : MonoBehaviour
         public float grabPrecognition = 0f;
         [Tooltip("An amount to multiply the velocity of any objects being thrown. This can be useful when scaling up the `[CameraRig]` to simulate being able to throw items further.")]
         public float throwMultiplier = 1f;
+        [Tooltip("Automatically scale the throw velocity up and down to match the scale of the `[CameraRig]`.")]
+        public bool throwScalesWithPlayer = true;
```

Comment:
but is there a need where people may want it to be different? It used to be the other way before and it was changed because under certain circumstances it was preferred the current way.

I'm just thinking is it better to make it all optional now?

The alternative is we make a dropdown that has "throw strategies" and they basically point to a script that calculates whatever you want it to, we build some default ones, but people can write their own if they want.

I can try doing a PR with the idea if you want?